framework_tvb/tvb/tests/framework/core/factory.py

```diff
@@ -154,8 +154,8 @@ def create_operation(test_user=None, test_project=None, operation_status=STATUS_
         adapter = ABCAdapter.build_adapter(algorithm)
         view_model = adapter.get_view_model_class()()
         view_model.data_file = "."
-        operation = Operation(test_user.id, test_project.id, algorithm.id,
-                              '{"gid": "' + view_model.gid.hex + '"}', status=operation_status)
+        operation = Operation(None, test_user.id, test_project.id, algorithm.id,
```

Comment:
Why not link the `view_model.gid` here ?

Reply:
it was a miss on my side, the gid should have been linked